e2e/windows/e2e-scenario.sh

```diff
@@ -12,38 +12,38 @@ collect-logs() {
     local retval
     retval=0
     mkdir -p $SCENARIO_NAME-logs
-    VMSS_INSTANCE_ID="$(az vmss list-instances --name $DEPLOYMENT_VMSS_NAME -g $MC_RESOURCE_GROUP_NAME | jq -r '.[0].instanceId')"
+    VMSS_INSTANCE_ID="$(az vmss list-instances --name $DEPLOYMENT_VMSS_NAME -g $MC_E2E_RESOURCE_GROUP_NAME | jq -r '.[0].instanceId')"
```

Comment:
Unify the format, e.g. E2E_MC_RESOURCE_GROUP_NAME?

Reply:
Thanks for the comment. Done.